Directory.Build.props

```diff
@@ -63,6 +65,12 @@
     <EnableNETAnalyzers>true</EnableNETAnalyzers>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <EnablePackageValidation>$(IsPackable)</EnablePackageValidation>
+    <!-- TODO Remove suppression once 6.0.0 is released. -->
+    <NoWarn>$(NoWarn);PKV006</NoWarn>
```

Comment:
Remove before merge.

Reply:
No, this needs removing after we push our `6.0.0` packages to NuGet.org (i.e. it becomes the baseline for `6.0.1` and later versions).

Reply:
#625